src/number_theory/modular.lean

```diff
@@ -323,10 +323,18 @@ by rw [←coe_re, coe_T_zpow_smul_eq, add_re, int_cast_re, coe_re]
 lemma im_T_zpow_smul (n : ℤ) : ((T^n) • z).im = z.im :=
 by rw [←coe_im, coe_T_zpow_smul_eq, add_im, int_cast_im, add_zero, coe_im]
 
-lemma re_T_smul : (T • z).re = z.re + 1 := by simpa using re_T_zpow_smul z 1
-lemma im_T_smul : (T • z).im = z.im := by simpa using im_T_zpow_smul z 1
-lemma re_T_inv_smul : (T⁻¹ • z).re = z.re - 1 := by simpa using re_T_zpow_smul z (-1)
-lemma im_T_inv_smul : (T⁻¹ • z).im = z.im := by simpa using im_T_zpow_smul z (-1)
+lemma re_T_smul : (T • z).re = z.re + 1 := by simp only [modular_T_smul, vadd_re, add_comm]
+lemma im_T_smul : (T • z).im = z.im := by simp only [modular_T_smul, vadd_im]
+lemma re_T_inv_smul : (T⁻¹ • z).re = z.re - 1 :=
```

Comment:
Can you insert spaces between the lemmas? In general, we don't skip lines only when the lemma and its proof fits on one line.